assets/css/style.css

```diff
@@ -0,0 +1,269 @@
+body{
+    background: #f4f5f6;
+}
+.main-container {
+    background: white;
+    padding-left: 25px;
```

Comment:
These two `padding` properties are overriding bootstrap's `padding` properties inside `.container` class. let me know if you did it on purpose

Reply:
yeah we did that on purpose because we needed smaller padding as opposed to that provided by bootstrap default